packages/atomic/src/components/insight/atomic-insight-interface/e2e/atomic-insight-interface.e2e.ts

```diff
@@ -0,0 +1,107 @@
+import {test, expect} from './fixture';
+
+const defaultInsightQuerySummaryText = 'Insights related to this case';
+
+test.describe('Atomic Insight Panel', () => {
+  test.beforeEach(async ({insightInterface}) => {
+    await insightInterface.load();
+    await insightInterface.hydrated.waitFor();
+  });
+
+  test('should load correctly', async ({insightInterface}) => {
+    await expect(insightInterface.insightInterface).toHaveClass('hydrated');
+  });
+
+  test.describe('components basics', () => {
+    test('parts should load correctly', async ({insightInterface}) => {
```

Comment:
The plan is to add to this tests other general checks later in separate PRs right? things like trying to perform a query, selecting a facet value, selecting a tab?..

Reply:
Selecting a tab is already there further down,

The others, selecting a facet, performing a query could be added later yes, the goal was to make sure to have at least basic smoke tests to cover if the component simply didn't load.

Let's plan to add those later, but they shouldn't be too hard to add.